src/io/GdmlGeometryMap.cc

```diff
@@ -69,7 +61,7 @@ const ImportElement& GdmlGeometryMap::get_element(elem_id element_id) const
  *
  * Can be used to preallocate storage for each thread for XS calculations
  */
-size_type GdmlGeometryMap::max_num_elements() const
+auto GdmlGeometryMap::max_num_elements() const -> size_type
 {
     size_type result = 0;
     for (const auto& key : matid_to_material_)
```

Comment:
Unrelated to your changes, but should this be `std::max(...)` below?

Reply:
Hah you’re right, I missed that in my first review of this class. I didn’t want to touch it because I think @stognini is working on it now for cutoffs. Once we get those in place we should probably simplify this class.